src/winconpty/winconpty.cpp

```diff
@@ -459,6 +479,21 @@ extern "C" HRESULT WINAPI ConptyClearPseudoConsole(_In_ HPCON hPC)
 }
 
 // Function Description:
+// - Tell the ConPTY about the state of the hosting window. This should be used
+//   to keep ConPTY's internal HWND state in sync with the state of whatever the
+//   hosting window is.
+// - For more information, refer to GH#12515.
+extern "C" HRESULT WINAPI ConptyShowHidePseudoConsole(_In_ HPCON hPC, bool show)
+{
+    const PseudoConsole* const pPty = (PseudoConsole*)hPC;
+    HRESULT hr = pPty == nullptr ? E_INVALIDARG : S_OK;
+    if (SUCCEEDED(hr))
+    {
+        hr = _ShowHidePseudoConsole(pPty, show);
+    }
+    return hr;

```

Comment:
This entire thing can be replaced with:
```cpp
return _ShowHidePseudoConsole((PseudoConsole*)hPC, show);
```

Same for the function above that uses `_ClearPseudoConsole` btw.